packages/obonode/obojobo-sections-assessment/components/full-review/index.js

```diff
@@ -36,13 +38,15 @@ class AssessmentReviewView extends React.Component {
 			allScoreDetails: attempts.map(a => a.scoreDetails)
 		})
 
-		const attemptReviewComponent = (attempt, isAHighestScoringNonNullAttempt) => {
+		const attemptReviewComponent = (attempt, isNonNullAttempt) => {
 			const date = new Date(attempt.completedAt)
 			const dateString = formatDate(date, "M/dd/yy 'at' h:mmaaaa")
 			const machineDateString = formatDate(date, "yyyy-MM-dd'T'HH:mm:ss.SSSxxx")
 			const ariaDateString = formatDate(date, "MMMM Do yyyy 'at' h:mmaaaa")
 			const numCorrect = AssessmentUtil.getNumCorrect(attempt.result.questionScores)
 			const numPossibleCorrect = AssessmentUtil.getNumPossibleCorrect(attempt.result.questionScores)
+			const isAHighestScoringNonNullAttempt =
+				isNonNullAttempt && attempt.assessmentScore === highestAttemptScore
```

Comment:
I don't think this will work in every case, since assessmentScore and attemptScore are two different types of scores. attemptScore will always be a percentage of what was right or wrong, but assessmentScore could be anything based on the assessment rubric.

Instead I think calling on `AssessmentUtil.getHighestAttemptsForModelByAssessmentScore` _should_ work to get the highest scoring attempt(s)

Reply:
Oh, I see that now. That makes sense.